src/Schema/MySQLSchemaManager.php

```diff
@@ -385,4 +387,35 @@ private function parseCreateOptions(?string $string): array
 
         return $options;
     }
+
+    private function buildColumnDefinition(array $tableColumn): string
```

Comment:
Not happy about that.

There is no native mysql syntax to show the create statement for a single column.